examples/hello_js/mos.yml

```diff
@@ -23,13 +23,13 @@ conds:
         - ["spi.miso_gpio", 19]
         - ["spi.sclk_gpio", 5]
         - ["spi.cs0_gpio", 15] # EPD
-        # - ["spi.cs1_gpio", 32] # SRAM - optional!
+        - ["spi.cs1_gpio", 32] # SRAM - optional!
```

Comment:
can we keep this file unchanged?
all examples are currently configured to use `SSD1675` and there's no necessity to change this